client/src/Game/Home.js

```diff
@@ -324,9 +333,50 @@ class Home extends Component {
 
     }
 
+    toggleTab(tabID) {
+        if (this.state.homeState !== tabID) {
+            this.setState({
+                homeState: tabID
+            });
+        }
+    }
+
+    renderTab(tabID) {//style={{marginLeft: "10%", marginRight:"10%"}}
+        return (
+            <Col>
+                <NavItem key={tabID}>
+
+                    <NavLink key={tabID}
+                             className={classnames({active: this.state.homeState === tabID})}
+                             onClick={() => {
+                                 this.toggleTab(tabID);
+                                 this.getGames();
+                             }}
+                    >
+                        {tabID}
+                    </NavLink>
+                </NavItem>
+            </Col>
+        );
+    }
+
+    renderTabContents(tabContents, tabID) {
+        return (
+            <Row>
+                <Col md={{size:10, offset:1}}>
+            <TabContent key={tabID} activeTab={this.state.homeState}>
+                <TabPane key={tabID} tabId={tabID}>
+                    {tabContents}
+                </TabPane>
+            </TabContent>
+                </Col>
+            </Row>
+        );
+    }
+
     render() {
-        console.log(this.state);
         this.updatePlayerNames();
+        this.getGames();
```

Comment:
So this isn't a good thing to do. This statement will cause an infinite loop, that continually makes calls to the back to get updated matches.